src/store/Store.ts

```diff
@@ -6,11 +6,18 @@ export interface ServiceGraphState {
   showNodeLabels: boolean;
 }
 
+export interface ServiceGraphDataState {
+  isLoading: boolean;
```

Comment:
You don't need your own individual `isLoading` indicator this is supposed to be global.